js/model.js

```diff
@@ -0,0 +1,27 @@
+
+var questionCount = 0;
+
+
+var createQuestions = function(questions) {
```

Comment:
The name of a class should start with upper case:(https://en.wikipedia.org/wiki/Naming_convention_(programming)#JavaScript)  
Class names should be nouns:
(http://www.oracle.com/technetwork/java/codeconventions-135099.html)